app/javascript/components/profile/relations.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div class="profile-relations">
+        <div class="profile-relations__title">
+            {{ $i18n.t('message.profile.relationsTitle') }}
+        </div>
+        <div class="profile-relations-users">
+          <div v-if="beingFetched || !recommendations" class="loading-icon"></div>
+          <vue-horizontal-list v-else :items="allRelationsBadges(recommendations['all'])" :options="horizontalListOptions">
+            <template v-slot:default="{item}">
+              <a
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 10 spaces but found 14 spaces                   vue/html-indent